src/routes.lisp

```diff
@@ -42,24 +42,22 @@
                                   :decorators (@rss))
     ((page :init-form 0 :parameter-type 'integer))
   (with-fail-handler (rss-feed :type 'web-view)
-    (let* ((board (parse-board board)))
-      (if (and board
-               (single-board-p))
+    (let ((board (parse-board board)))
+      (if (and board (single-board-p))
           (render-error-page "This instance only has a single board." 404)
           (progn
             (get-request-validity-check :board board :page page)
             (let* ((posts (select-posts 30 (* page 30) :board board))
-                   (sorted-posts (sort posts
-                                       #'sort-posts-by-id)))
+                   (sorted-posts (sort posts #'sort-posts-by-id)))
               (with-output-to-string (s)
                 (with-rss2 (s :encoding "utf-8")
                   (rss-channel-header "nmebious" *web-url*
                                       :description "monitoring the wired")
                   (dolist (item sorted-posts)
-                    (let* ((data (cassoc :data item))
```

Comment:
let* is used for when you want to reference the previous let bindings in your let (as the regular let cannot do this).
example:
`
(let* ((a 5)
         (b (* a 2))
  b)
`
let* is actually just a macro which expands to.
`
(let ((a 5))
  (let ((b (* a 2)))
     b))
`
and doing all this expansion is unnecessary when you don't need it